src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellPageRendererTracker.cs

```diff
@@ -782,7 +782,7 @@ void SearchButtonClicked(object sender, EventArgs e)
 
 		void SetSearchBarIcon(UISearchBar searchBar, ImageSource source, UISearchBarIcon icon)
 		{
-			source.LoadImage(source.FindMauiContext(), image =>
+			source.LoadImage(source.FindMauiContext(true), image =>
```

Comment:
```suggestion
			source.LoadImage(source.FindMauiContext() ?? MauiContext, image =>
```

Reply:
Mnn, the `FindMauiContext `extension method uses the `MauiContext `of the App if it is null, but couldn't the `MauiContext `property in this context be null?